.github/CODEOWNERS

```diff
@@ -591,6 +591,9 @@
 # PRLabel: %Mixed Reality
 /sdk/objectanchors/                                                @crtreasu @rgarcia @JoshLove-msft
 
+# PRLabel: %Open AI
```

Comment:
We'll need to add this to the [Fabric Bot rules](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/fabricbot.json) if we want the PR label to be applied.     We should also add a "Service Attention" slug and bot rule.

Happy to walk through that with you, if you'd like.